app/editor/src/features/content/papers/Papers.tsx

```diff
@@ -41,138 +41,126 @@ export interface IPapersProps extends React.HTMLAttributes<HTMLDivElement> {}
-  const [delayProcessing, setDelayProcessing] = React.useState<boolean>(false);
+  const selectedIds = selected.map((i) => i.id.toString());
 
-  hub.useHubEffect(MessageTargetName.ContentAdded, (message) => {
-    if (message.ownerId === userId) {
-      setContentUpdatesQueue((queue) => [...queue, message]);
-    }
-  });
+  const calcTotals = React.useCallback(
+    (items: IContentSearchResult[], filter: IContentListFilter) => {
+      const topStories = items.filter((i) => i.isTopStory).length;
+      const commentary = items.filter((i) => i.isCommentary).length;
+      const featuredStories = items.filter((i) => i.isFeaturedStory).length;
+      const published = items.filter((i) =>
+        [ContentStatusName.Publish, ContentStatusName.Published].includes(i.status),
+      ).length;
+      if (
+        !filter.topStory &&
+        !filter.commentary &&
+        !filter.featuredStory &&
+        !filter.onlyPublished
+      ) {
+        setTotals((totals) => ({
+          ...totals,
+          topStories,
+          commentary,
+          featuredStories,
+          published,
+        }));
+      }
+      if (filter.topStory) setTotals((totals) => ({ ...totals, topStories }));
+      if (filter.commentary) setTotals((totals) => ({ ...totals, commentary }));
+      if (filter.featuredStory) setTotals((totals) => ({ ...totals, featuredStories }));
+      if (filter.onlyPublished) setTotals((totals) => ({ ...totals, published }));
+    },
+    [],
+  );
 
   hub.useHubEffect(MessageTargetName.ContentUpdated, (message) => {
-    setContentUpdatesQueue((queue) => [...queue, message]);
+    getContent(message.id)
+      .then((content) => {
+        if (content) {
+          setCurrentResultsPage((page) => {
+            const newPage = {
+              ...page,
+              items: page.items.map((q) =>
+                q.id === content.id ? castContentToSearchResult(content) : q,
+              ),
+            };
+            if (
+              filter.topStory ||
+              filter.commentary ||
+              filter.featuredStory ||
+              filter.onlyPublished
+            ) {
+              newPage.items = newPage.items.filter((i) =>
+                filter.topStory
+                  ? i.isTopStory
+                  : true && filter.commentary
+                  ? i.isCommentary
+                  : true && filter.featuredStory
+                  ? i.isFeaturedStory
+                  : true && filter.onlyPublished
+                  ? [ContentStatusName.Publish, ContentStatusName.Published].includes(i.status)
+                  : true,
+              );
+            }
+            calcTotals(newPage.items, filter);
+            return newPage;
+          });
+        }
+      })
+      .catch(() => {});
   });
 
+  // Stores the current page
   React.useEffect(() => {
-    if (!isProcessingMessages && !delayProcessing && contentUpdatesQueue?.length) {
```

Comment:
I ripped out all of this queue stuff, as it was complicated the simple update process.  The only thing it probably was solving was sometimes more than single request is made due to updates.  This can lead to the content being stale.  I haven't seen the error occur, but I think it can.